### PR TITLE
Filter rates indi

### PR DESCRIPTION
**Pull request by EwoudSmeur**
This PR makes the filtering of body rates in the INDI controller use the first order filter in the filters folder. Now only the cutoff frequency needs to be specified to activate the filter. I updated all airframe files that I could see were using the filtering.

### CODE REVIEW

File: conf/airframes/ENAC/hoops_gen_ap.xml [hunk 1]
```diff
@@ -206,9 +206,9 @@
     <define name="G1_R" value="0.0022"/>
     <define name="G2_R" value="0.20"/>
 
-    <define name="FILTER_ROLL_RATE" value="TRUE"/>
-    <define name="FILTER_PITCH_RATE" value="TRUE"/>
-    <define name="FILTER_YAW_RATE" value="FALSE"/>
+    <!--unit: rad/s-->
+    <define name="STABILIZATION_INDI_FILT_CUTOFF_P" value="100.0"/>
```
Comment (fvantienen) [inline conf/airframes/ENAC/hoops_gen_ap.xml:210]: I think these are incorrect

**Comment by gautierhattenberger** [comment]
Why sometimes the `FILT_CUTOFF_R` is removed and sometimes replaced by `FILT_CUTOFF_RDOT` ?

File: conf/airframes/ENAC/hoops_gen_ap.xml (hunk 1 above)
Comment (EwoudSmeur) [reply conf/airframes/ENAC/hoops_gen_ap.xml]: :sweat_smile:

**Comment by EwoudSmeur** [comment]
> Why sometimes the `FILT_CUTOFF_R` is removed and sometimes replaced by `FILT_CUTOFF_RDOT` ?

The naming before was confusing/incorrect. There are two filters that are used: one for the angular acceleration and one for the angular rates, because of different noise levels due to the derivative. Before we only had the filter for the angular acceleration and then R referred to angular acceleration around body Z.

**Review by gautierhattenberger — APPROVED** [review]

**Review by fvantienen — APPROVED** [review]

**Comment by fvantienen** [comment]
I can't merge it because of semaphore failing, but for me it is fine to merge

**Pull request merged**